certificate_agent.py

```diff
@@ -98,7 +98,7 @@ def main():
                     template_pdf,
                     aws_id=args.aws_id,
                     aws_key=args.aws_key,
-                    long_course=course_name,
+                    long_course=course_name.encode('utf-8'),
```

Comment:
Nope.
The decode method was already called inside each `generate`_WHATEVER_`certificate` for each different version of certificate . Actually new `generate` methods use decoding inside the function and old ones (v1,v2) are corrected in this PR  so I'm assuming that long_course should be an unicode object for all this methods